skyproj/skyproj.py

```diff
@@ -101,8 +102,25 @@ def __init__(self, ax=None, projection_name='cyl', lon_0=0, gridlines=True, cele
         self._wrap = (lon_0 + 180.) % 360.
         self._lon_0 = self.projection.proj4_params['lon_0']
 
+        self._full_sky = False
+        self._full_sky_extent = [
+            lon_0 - 180.0,
+            lon_0 + 180.0,
+            -90.0 + self._pole_clip,
+            90.0 - self._pole_clip
+        ]
+
+        self._full_sky = False
```

Comment:
Why is `self._full_sky` begin set to `False` again? This was done 8 lines before.

Reply:
The lat full_sky_extent seems wrong for HammerSkyproj, but this may be a weird thing with this projection:
```
smap = skyproj.HammerSkyproj()
smap.scatter([0, 90], [0, 45])
print(smap._full_sky, smap._full_sky_extent)
>>> True (179.7883460590184, -179.7883460590184, -2.897600013847956, 2.897600013847956)
```